### ISSUE TITLE
NuGet

### ISSUE THREAD

**Issue by t1st3** [damieng/silk-companion#3]
@damieng I recently managed to get some of my packages available on [NuGet](https://www.nuget.org/), which is a package manager, mainly geared towards .NET packages.

If you're OK with publishing silk-companion on NuGET, I'd be happy to help / give some guidelines.

The process is not straightforward, so this discussion will stand as an entry point for all the steps needed in order to achieve publishing on NuGet.

**Comment by damieng** [comment]
We can do that - I actually have a couple of packages on NuGet already -
CSharpAnalytics and ElasticLINQ but haven't considered how to get static
assets like this packaged up there correctly.

**Comment by t1st3** [comment]
Well, it will be helpful!

The main problem with NuGet for me is that I'm not a Windows user; I'm definitely on Linux side. This makes publishing on NuGet kind of cumbersome, since I would normally require nuget.exe (Windows-specific) to publish packages on NuGet, which is not Linux-compatible.

I found a way to circumvent this.
There's a service called [MyGet](https://www.myget.org/), which would work like Travis-CI in some way: on each modification on Github, the project is cloned from Github to MyGet, and then, MyGet compiles a NuGet package automatically from Github source (provided we give them the way to do it). Once the NuGet package is built on MyGet, we just have have to click on a "PUSH" button, and the package is sent to [nuget.org](https://www.nuget.org/).

Like I said, it's not straightforward, but kind of easily achievable for a Linux user, without the need of nuget.exe.

I successfully did it for `famfamfam` packages. But maybe you'd have an easier way to publish, since it's not your first time with this.

thanks @supergibbs for his work on Bootstrap, which gave me all these ideas; I would not have found them by myself!

Thoughts welcome!
